core/src/main/scala/akka/kafka/KafkaConsumerActor.scala

```diff
@@ -136,7 +136,7 @@ private[kafka] class KafkaConsumerActor[K, V](settings: ConsumerSettings[K, V])
           // this is invoked on the thread calling consumer.poll which will always be the actor, so it is safe
           val duration = System.nanoTime() - startTime
           if (duration > settings.commitTimeWarning.toNanos) {
-            log.warning("Kafka commit took longer than `commit-time-warning`: {} ms", duration)
+            log.warning("Kafka commit took longer than `commit-time-warning`: {} ns", duration)
```

Comment:
Not quite convinced ns granularity is useful here, wouldn't truncating to ms, or pretty-printing to biggest possible unit (I think we have some util for that in Akka) be better?

Reply:
do you mean `toCoarsest`? I think that will not help here, because that will always return nanos in this case. I've changed the code so it will convert to millis now.